src/Domain/FunctionsOfTime/Tags.hpp

```diff
@@ -76,17 +76,24 @@ struct FunctionsOfTime : db::SimpleTag {
       const std::optional<std::map<std::string, std::string>>&
```

Comment:
is it possible to have the user specify the max deriv for the map? I'm a bit worried that if the max deriv isn't 2 or 3 you get a silent failure. If the user specified the max deriv you can have an `if-else` and then add an `ERROR` to the reader function since it then must succeed. That seems quite a bit safer to me :Shrug: